docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   db:
     container_name: postgis
-    image: postgis/postgis:latest
```

Comment:
x86 image works fine on Apple Silicon with emulation as far as I can see.

Reply:
I am running into issues when I do that and I can see there are github issues open on postgis repo regarding this, but its okay, I will just manually make the change locally when I need to run it.